project1/application.py

```diff
@@ -30,12 +35,20 @@ def index():
 def register():
     
     if (request.method=="POST"):
+        data.query.all()
         name = request.form.get("name")
+        print(name)
         pswd = request.form.get("Password")
         print(pswd)
-        return render_template("login.html",name=name)
-
-    return render_template("register.html")
+        
+        contact = request.form.get("ContactNumber")
+        print(contact)
+        regist = data(username=name,ContactNumber=contact)
```

Comment:
contact number doesnt seem right, better to save uname and passwrd

Reply:
okay madam i will change..